stable/fluent-bit/values.yaml

```diff
@@ -24,15 +24,20 @@ metrics:
 trackOffsets: false
 
 backend:
-  type: forward
+  file:
+    enabled: false
+  stdout:
+    enabled: true
   forward:
+    enabled: false
```

Comment:
This goes against the recommendation that fluent-bit should be used as a log shipper. It should default to true.